Editor/Resources/EditorWindow/Pages/AnywherePage.uxml

```diff
@@ -0,0 +1,100 @@
+            <ui:Label name="AnywherePageTitle" class="page__title" text="GameLift Anywhere"/>
+            <ui:Label name="AnywherePageDescription" text="Amazon GameLift Anywhere fleets use your existing infrastructure to host game servers while Amazon GameLift manages your matchmaking and queues.&#10;You can integrate Anywhere and Managed EC2 fleets in matchmaker and game session queues to manage matchmaking and game placement."/>
+        </ui:VisualElement>
+        <ui:VisualElement class="divider"/>
+        <ui:Foldout name="AnywherePageIntegrateTitle" class="foldout separator separator--small" text="Integrate GameLift with your game client">
+            <ui:Label name="AnywherePageIntegrateDescription" text="Add the Amazon GameLift server SDK to your game server and add code to communicate with the GameLift hosting service. If you&apos;re working with the Amazon GameLift sample game, this integration is already done."/>
+            <ui:Label name="AnywherePageIntegrateServerLink" class="external-link" text="Setup Amazon GameLift on a backend service \uf08e"/>
+            <ui:Label name="AnywherePageIntegrateClientLink" class="external-link" text="Setup Amazon GameLift client \uf08e"/>
+        </ui:Foldout>
+        <ui:Foldout name="AnywherePageCreateFleetTitle" class="foldout" text="Connect to Anywhere Fleet">
+            <ui:VisualElement name="AnywherePageFleetName" class="form-row">
+                <ui:Label name="AnywherePageConnectFleetName" class="form-row__label" text="Fleet Name"/>
+                <ui:VisualElement class="form-row__input">
+                    <ui:TextField name="AnywherePageCreateFleetInput" class="text-input" value="GameName-AnywhereFleet"/>
+                    <ui:Label name="AnywherePageCreateFleetHint" class="input-hint" text="Fleet Name must have 1–1024 characters. Valid characters are A-Z, a-z, 0-9, _ and - (hyphen)"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+            <ui:VisualElement class="form-row">
+                <ui:VisualElement class="form-row__input form-row__input--no-label">
+                    <ui:Button name="AnywherePageCreateFleetButton" class="button button--primary" text="Create Anywhere Fleet"/>
+                    <ui:Button name="AnywherePageCreateFleetCancelButton" class="button" text="Cancel"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+        </ui:Foldout>
+        <ui:Foldout name="AnywherePageConnectFleetTitle" class="foldout" text="Connect to Anywhere Fleet">
+            <ui:VisualElement name="AnywhereFleetName" class="form-row">
+                <ui:Label name="AnywherePageConnectFleetLabel" class="form-row__label" text="Fleet Name"/>
+                <ui:DropdownField name="AnywherePageConnectFleetDropdown" class="form-row__input dropdown"/>
+            </ui:VisualElement>
+            <ui:VisualElement name="AnywherePageConnectFleetID" class="form-row form-row__hidden">
```

Comment:
`form-row__hidden` doesn't appear to be used anywhere. Was this meant to be `foldout--hidden` or something else?

Reply:
removed 477d6e81507726d4414c14f7abc96df2e315dd36